draft-ietf-rtcweb-jsep.xml

```diff
@@ -556,6 +556,82 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
       </section>
 
+      <section anchor="sec.imageattr" title="Video Size Negotiation">
+        <t>Video size negotiation is the process through which a receiver can
+          use the "a=imageattr" SDP attribute <xref target="RFC6236"/> to indicate
+          what video frame sizes it is capable of receiving. A receiver may have
+          hard limits on what its video decoder can process, or it may wish to
+          constrain what it receives due to application preferences, e.g. a
+          specific size for the window in which the video will be displayed.
+        </t>
+
+        <section title="Creating an imageattr Attribute">
+          <t>In order to determine the limits on what video resolution a
+            receiver wants to receive, it will intersect its decoder hard
+            limits with any mandatory constraints that have been applied to
+            the associated MediaStreamTrack. For the answerer, these mandatory
+            constraints can be applied to the remote MediaStreamTracks that
+            are created by a setRemoteDescription call, and will affect the
+            output of the ensuing createAnswer call. For the offerer, since it
+            does not know about any remote MediaStreamTracks until it receives
+            theanswer, this implies that it can only set these constraints after
+            the initial offer-answer exchange is complete, and as a result, a
+            new offer-answer is needed to communicate them.</t>
```

Comment:
A sends offer to B. Lets say it has 

   a=imageattr:97 send * recv *

B wants to limit it was it receives to 360p and sends back an pr-answer with 
    
    a=imageattr:* recv [x=[16:640],y=[16:360],q=1.0]

and A starts sending 360p to B when it gets this pr-answer. 

I'm not seeing why an 2nd offer/answer is needed. You do an answer of pr-answer before the constraint can take place but given an pr-answer will work, I don't think it is right to say that the offer-answer needs to complete. 

It seems like replacing "the initial offer-answer exchange is complete, and as a result, a new offer-answer is needed to communicate them"  with "receiving the initial answer or pr-answer. " would fix this.